requirements.txt

```diff
@@ -1 +1,2 @@
 librosa>=0.4.3
+tensorflow>=0.12
```

Comment:
Maybe tensorflow-gpu would be more reasonable?
Most of CPU tensorflow users might not want to train this model anyway.